app/controllers/subscriptions_controller.rb

```diff
@@ -18,7 +18,7 @@ class SubscriptionsController < CrudController
 
   alias mailing_list parent
 
-  def index # rubocop:disable Metrics/MethodLength there are a lof of formats supported
+  def index # rubocop:disable Metrics/AbcSize,Metrics/MethodLength there are a lof of formats supported
```

Comment:
Ist dieses zusätzliche rubocop ignore noch nötig? An dieser Methode hast du ja schlussendlich nichts geändert.

Reply:
RuboCop hat sich beschwert, dass es zu komplex ist. Da es durch die Anzahl der Formate nicht wirklich einfacher wird, habe ich es hinzugefügt. 

Seit mein Editor mir das inline anzeigt, gehe ich das immer direkt an. Vielleicht ist es vorher einfach keinem aufgefallen oder die Defaults haben sich geändert. Die AbcSize ist IIRC nicht in der `rubocop-must.yml` drin, daher ist das eher die langsame Konvergenz zu einem sauberen Setup. :-)